src/jit/importer.cpp

```diff
@@ -2786,10 +2786,12 @@ GenTreePtr      Compiler::impInitializeArrayIntrinsic(CORINFO_SIG_INFO * sig)
 #endif
         )
     {
+#if COR_JIT_EE_VERSION > 460
```

Comment:
@jkotas Sorry, I've seen the COR_JIT_EE_VERSION check in your code but I forgot to add it here. Thanks!